presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -1150,6 +1150,79 @@ private void testInsertPartitionedTableExistingPartition(Session session, HiveSt
     }
 
     @Test
+    public void testPartitionPerScanLimit()
+            throws Exception
+    {
+        for (TestingHiveStorageFormat storageFormat : getAllTestingHiveStorageFormat()) {
```

Comment:
I am not sure if we want to test it for all storage format. Seems a bit overkill?